src/types/long.js

```diff
@@ -0,0 +1,860 @@
+import { typeName, setUpInheritance } from '../type';
+import { remapToJs } from '../ffi';
+import { pyCheckArgs, checkString, checkNumber } from '../function/checks';
+import { func } from '../function'
+import { TypeError, ValueError, NotImplementedError } from '../errors';
+import { NotImplemented } from './object';
+import { asnum$ } from '../builtin';
+import { isIndex, asIndex } from '../misceval';
+import { numtype } from './numtype';
+import { str } from './str';
+import { int_, str2number } from './int';
+import { float_ } from './float';
+import { tuple } from './tuple';
+import { true$, false$ } from '../constants';
+import { bool } from './bool'
+
+import biginteger from 'big-integer';
```

Comment:
@rixner this is the package I was talking about.

Reply:
@albertjan I'm glad we're on the same page.  I suppose we'd need to discuss which package makes the most sense, but that's a future discussion...

Reply:
Yeah I picked one that was actively maintained had a lot of downloads and was a polyfill for `BigInt` I think it's actually a future version of what we use :smile: